src/App.tsx

```diff
@@ -1,13 +1,46 @@
 import { Route, Routes } from "react-router-dom";
 import Home from "./pages/Home/Home";
 import "./styles/main.scss";
+import { useState } from "react";
 import Nav from "./components/Nav/Nav";
+import Login from "./pages/Login/Login";
+import Register from "./pages/Register/Register";
 
 const App = () => {
+  const [userId, setUserId] = useState<string>("");
+  const [email, setEmail] = useState<string>("");
+  const [password, setPassword] = useState<string>("");
+
+  console.log(userId);
```

Comment:
Can you remove the `console.log` please?

Reply:
We found that this is the only way to utilise the userId variable so that we can push our changes, unless you have another suggestion?

Reply:
the console log has to stay at the moment otherwise the linter will fail the build. Put it back in for now please

Reply:
We ended up using it in the login page to display at the bottom following a successful login. Is that alright?

Reply:
Yeah that is cool, I didn't think about the linting warning